addons/field_rations/XEH_preInit.sqf

```diff
@@ -14,12 +14,6 @@ PREP_RECOMPILE_END;
 GVAR(thirstModifiers) = [];
 GVAR(hungerModifiers) = [];
 
-private _cache = call (uiNamespace getVariable [QGVAR(cacheP3Ds), {ERROR("no cache")}]); // cache built at preStart
-// List of p3d water sources (case sensitive) for objects that report `typeOf == ""`
-GVAR(waterSourceP3Ds) = _cache select 0;
-// List of refill action offsets corresponding to the p3ds in the array above
-GVAR(waterSourceOffsets) = _cache select 1;
-
 // Custom Arsenal Tab
```

Comment:
We could make a copy like
`GVAR(cacheP3Ds) = +(uiNamespace getVariable QGVAR(cacheP3Ds));`
so it's editable by mission makers/other mods at runtime